clusterloader2/testing/experimental/storage/pod-startup/persistent-volumes/config.yaml

```diff
@@ -0,0 +1,142 @@
+# ASSUMPTIONS:
+# TODO: Assumptions
+
+# Constants
+  {{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+  {{$DEFAULT_NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
+
+# Variables
+  {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 50}}
+  {{$PVC_PER_POD := DefaultParam .PVC_PER_POD 2}}
+  {{$ENABLE_CHAOSMONKEY := DefaultParam .ENABLE_CHAOSMONKEY false}}
+  {{$TEST_NAME := DefaultParam .TEST_NAME "generic-test"}}
+  {{$PVC_SIZE := DefaultParam .PVC_SIZE "8Gi"}}
+  {{$PER_PVC_BOUND_SLO := DefaultParam .PVC_BOUND_SLO 2}}
+  {{$POD_STARTUP_SLO := DefaultParam .POD_STARTUP_SLO 100}}
+  {{$OBJ_CREATION_QPS := DefaultParam .OBJ_CREATION_QPS 10}}
+
+# Calculated
+  {{$minNamespaces := DivideInt .Nodes $DEFAULT_NODES_PER_NAMESPACE}}
+  {{$namespaces := MaxInt $minNamespaces 1}}
+  {{$nodesPerNamespace := DivideInt .Nodes $namespaces}}
+  {{$podsPerNamespace := MultiplyInt $nodesPerNamespace $PODS_PER_NODE}}
+  {{$pvcsPerNamespace := MultiplyInt $podsPerNamespace $nodesPerNamespace}}
+  {{$totalPVCs := MultiplyInt $pvcsPerNamespace $namespaces}}
+  {{$PVCBoundTime := MultiplyInt $totalPVCs $PER_PVC_BOUND_SLO | MaxInt 60}}
+
+name: pv-storage-{{$TEST_NAME}}
+automanagedNamespaces: {{$namespaces}}
+tuningSets:
+  - name: Sequence
+    parallelismLimitedLoad:
+      parallelismLimit: 1
+  - name: UniformQPS
+    qpsLoad:
+      qps: {{$OBJ_CREATION_QPS}}
+  {{if $ENABLE_CHAOSMONKEY}}
+chaosMonkey:
+  nodeFailure:
+    failureRate: 0.01
+    interval: 1m
+    jitterFactor: 10.0
+    simulatedDowntime: 10m
+  {{end}}
+steps:
+  - measurements:
+      - Identifier: APIResponsiveness
+        Method: APIResponsiveness
+        Params:
+          action: reset
+      - Identifier: APIResponsivenessPrometheus
+        Method: APIResponsivenessPrometheus
+        Params:
+          action: start
+      - Identifier: TestMetrics
+        Method: TestMetrics
+        Params:
+          action: start
+          nodeMode: {{$NODE_MODE}}
+  # Create PVCs
```

Comment:
I'm wondering if we really need this file.
This test is extremely similar to: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/testing/experimental/storage/pod-startup/ephemeral-volumes/cluster-wide/config.yaml

The only difference is that instead of creating ephemeral storage we create PVCs and wait for them.
But it should be possible to unify them (if you wait for PVC in ephemeral-storage scenario, it will be no-op anyway).

So instead of having two separate files to maintain, I would much rather prefer unifying those two together - I don't think the differences justify having a separate one.

@mucahitkurt - as the one who added the test I mentioned above.

Reply:
I would be happy to merge the two configs (in a separate PR). There definitely are similarities although I think the PV config will be a little bit more config due to the WaitForBoundPVCs and such.

Reply:
I think the WaitForBoundPVCs is the only difference. And just adding it won't break ephemeral volumes, because simple there won't be anything to wait for there.
So unless I'm not missing anything, I'm against having two separate configs and merging them later - we should just do the right thing from the beginning.

Reply:
I would much prefer the base config be the one I wrote. There is a difference in that my config has an option to scale automatically based on # of nodes and overrides is used for different test variations (defined by the KEP), whereas @mucahitkurt’s work operates similar to configs for load and density where overrides is used to define the size of the cluster.

Its ok imo to not merge it yet, but it would take some time to work together and refactor how the configs work. Idealy, it should be the same base woth maybe a flag for pv vs ephemeral.

Reply:
> @mucahitkurt’s work operates similar to configs for load and density where overrides is used to define the size of the cluster.

Sorry If I got it wrong, but actually I don't use overrides for the cluster size, e.g. 1_node and cluster_wide case, I use different configs under `1_node` and `cluster-wide`, I started with one config but got feedback on the related PR to separate the configs for those cases for the sake of simplicity over duplication. But of course we can think that cluster-wide config already covers the 1 node case, as you already did.

Reply:
I'm not sure if it will cause to some limitations, but may be it would be better if we can get rid of root `ephemeral-volumes` directory and use the PVC as a different volume type

Reply:
> I'm not sure if it will cause to some limitations, but may be it would be better if we can get rid of root ephemeral-volumes directory and use the PVC as a different volume type

Yes - we probably should.

> I would much prefer the base config be the one I wrote. There is a difference in that my config has an option to scale automatically based on # of nodes and overrides is used for different test variations (defined by the KEP), whereas @mucahitkurt’s work operates similar to configs for load and density where overrides is used to define the size of the cluster.

I'm not trying to say you can't modify @mucahitkurt config - we definitely need to. And I'm completely fine with modify something there.
I'm also fine with replacing the one added by @mucahitkurt with this one (though I need to review it more carefully at that point). I just don't see any reason to have both at this point.

Reply:
With https://github.com/kubernetes/perf-tests/pull/688 merged, I think this file is no longer needed - just that one requires changes. Can you please update this PR?

Reply:
ack